keyvi/include/keyvi/dictionary/fsa/comparable_state_traverser.h

```diff
@@ -168,6 +168,15 @@ class ComparableStateTraverser final {
 
   size_t GetOrder() const { return order_; }
 
+  /**
+   * Set the minimum weight states must be greater or equal to.
+   *
+   * Only available for WeightedTransition specialization.
```

Comment:
I had it like this in an earlier version, but I reverted, because it broke completions on dictionaries without weights (test case should exist, so it is easy to find out).

I will try this once more and make a new PR if it doesn't break this case.

So many changes, at some point I lost track.